ownCloud/Resources/Info.plist

```diff
@@ -29,8 +33,14 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>NSAppleMusicUsageDescription</key>
+	<string>This permission is needed by the ownCloud app to upload media files to your server.</string>
```

Comment:
Sorry for not thinking of this again, but mentioning "ownCloud app" might be a problem for branding.

"This permission is needed for uploading media files to your server." ?